Tests/DatabaseServiceTests/Repositories/UserRepositoryDeleteTests.cs

```diff
@@ -0,0 +1,97 @@
+﻿using DataBaseService.Database;
+using DataBaseService.Database.Models;
+using DataBaseService.Mappers;
+using DataBaseService.Repositories;
+using Kernel.CustomExceptions;
+using Microsoft.EntityFrameworkCore;
+using NUnit.Framework;
+using System;
+using System.Linq;
+
+namespace DatabaseServiceTests
+{
+    public class UserRepositoryDeleteTests
+    {
+        UserRepository userRepository;
+        DbUserCredential dbUserCredential;
+        UserMapper mapper;
+        Guid userId;
+        Guid id;
+
+        [SetUp]
+        public void Initialize()
+        {
+            mapper = new UserMapper();
+            userId = Guid.NewGuid();
+            id = Guid.NewGuid();
+
+            dbUserCredential = new DbUserCredential
+            {
+                Id = id,
+                UserId = userId,
+                Email = "example@gmail.com",
+                PasswordHash = "hashpassword",
+                IsActive = true
+            };
+        }
+
+        [Test]
+        public void DeleteUserOk()
+        {
+            var options = new DbContextOptionsBuilder<TPlatformDbContext>()
+                .UseInMemoryDatabase(databaseName: "DeleteUserOkey")
+                .Options;
+
+            using var dbContext = new TPlatformDbContext(options);
+            var userRepository = new UserRepository(mapper, dbContext);
```

Comment:
So i guess here you wanted to use not `var` but that variable

Reply:
fixed